playwright/tests/page-objects/inscription-view.ts

```diff
@@ -0,0 +1,49 @@
+import { Locator, Page, expect } from '@playwright/test';
+import { ViewData } from './view';
+import { IFrameView } from './iframe-view';
+
+export class InscriptionView extends IFrameView {
+  constructor(page: Page) {
+    const outputViewData: ViewData = {
+      tabSelector: 'li.action-item:has-text("IVY")',
+      viewSelector: 'body > div > div:not([data-parent-flow-to-element-id]) >> visible = true'
```

Comment:
what is this data-parent-flow...?

Reply:
that is the reference to the nested `.editor-instance` (vscode) element. only the process editor view has such a reference :D.